appsody deploy --dryrun fails

:\Users\appsody\.appsody\extract\express]
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0xa0 pc=0x7695dd]

goroutine 1 [running]:
os/exec.(*Cmd).Wait(0x0, 0x6, 0x6)
        /usr/local/go/src/os/exec/exec.go:466 +0x2d
github.com/appsody/appsody/cmd.RunDockerCommandAndWait(0xc00010c120, 0x6, 0x6, 0x970049, 0x6, 0x5, 0x5)
        /Users/sesadmin/go/src/github.com/appsody/appsody/cmd/docker_commands.go:26 +0x92
github.com/appsody/appsody/cmd.DockerBuild(...)
        /Users/sesadmin/go/src/github.com/appsody/appsody/cmd/docker_commands.go:17
github.com/appsody/appsody/cmd.glob..func1(0xd9bd00, 0xc00005d3b0, 0x0, 0x1, 0x0, 0x0)
        /Users/sesadmin/go/src/github.com/appsody/appsody/cmd/build.go:85 +0x5b7
github.com/appsody/appsody/vendor/github.com/spf13/cobra.(*Command).execute(0xd9bd00, 0xc00005d370, 0x1, 0x1, 0xd9bd00, 0xc00005d370)
        /Users/sesadmin/go/src/github.com/appsody/appsody/vendor/github.com/spf13/cobra/command.go:762 +0x46c
github.com/appsody/appsody/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xd9c420, 0x86c4e5, 0xd9c420, 0xc000095f88)
appsody build --dryrun
The problem with dryrun is because it doesn't skip the exec.Wait()  


Stack is as follows
      /Users/sesadmin/go/src/github.com/appsody/appsody/vendor/github.com/spf13/cobra/command.go:852 +0x2f3
github.com/appsody/appsody/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        /Users/sesadmin/go/src/github.com/appsody/appsody/vendor/github.com/spf13/cobra/command.go:800
github.com/appsody/appsody/cmd.Execute(0xa17720, 0x5)
        /Users/sesadmin/go/src/github.com/appsody/appsody/cmd/root.go:153 +0x5e
main.main()
        /Users/sesadmin/go/src/github.com/appsody/appsody/main.go:26 +0x40

--- a/cmd/docker_commands.go
+++ b/cmd/docker_commands.go
@@ -23,6 +23,11 @@ func RunDockerCommandAndWait(args []string, logger appsodylogger) error {
 	if err != nil {
 		return err
 	}
+	if dryrun {
+		Info.log("Dry Run - Skipping : cmd.Wait")
+
+		return nil
+	}
 	return cmd.Wait()
 
 }

